src/components/four-range-control/index.js

```diff
@@ -300,6 +360,8 @@ FourRangeControl.defaultProps = {
 	placeholderLeft: '',
 	initialPosition: '',
 
+	splitMode: undefined,
```

Comment:
I feel that `splitMode` doesn't describe best what our intention is. I'd recommend using `vhMode` instead because it shows clearly from the name that it's a "vertical-horizontal" mode